relay-server/src/actors/store.rs

```diff
@@ -1136,24 +1133,50 @@ struct CheckInKafkaMessage {
     retention_days: u16,
 }
 
-#[derive(Debug, Serialize)]
-struct SpanKafkaMessage<'a> {
-    /// Time at which the event was received by Relay. Not to be confused with `start_timestamp_ms`.
-    start_time: u64,
+#[derive(Debug, Deserialize, Serialize)]
+struct SpanKafkaMessage {
+    #[serde(skip_serializing)]
+    start_timestamp: f64,
+    #[serde(rename(deserialize = "timestamp"), skip_serializing)]
+    end_timestamp: f64,
+
+    description: String,
```

Comment:
It should be possible to make the fields borrowing, i.e. `&'de str` instead of `String`, `&'de RawValue` instead of `Value`.